README.md

```diff
@@ -27,16 +27,45 @@ Or install it yourself as:
       environment: "production",
       application: "ror"
     }
-    logger = Hivelog::Logger.new(:stdout, @labels)
+
     options = {
       event: event,
       user: user,
       group: group,
       tags: tags
     }
-    logger.debug("deb ya", options)
+
+    Hivelogger = Hivelog::Logger.new(:stdout, @labels)
+
+    Hivelogger.debug("deb ya", options)
+    Hivelogger.warn("warning message", options)
+    Hivelogger.info("information message", options)
+    Hivelogger.error("error message", options)
+```
+
+Each log is a sentence. A sentence always starts with capitalization and ends with a period.
+
+
+Hivelog's schema should respect ECS definitions for 2 mains reasons:
+- To avoid ES index mapping conflits
+- More readable for search and analytics
+
+You can find Guideline of ECS version 1.7 (current version) [here](https://www.elastic.co/guide/en/ecs/1.7/ecs-guidelines.html).
+
+Please scope with `tags` (e.g.: `donations`, `payment`, `recurly`, ...). It will help for filtering.
+
+```ruby
+    options = {
+      tags: ['memberships', 'payment'],
+    }
+
+    Hivelogger.info("information message", options)
 ```
 
```

Comment:
For each log should contain mandatory fields `event.action` and `event.category`, that will explicitly add keywords for search & filtering. I think it's more strict than tags.